src/lambdas/get-app-theme-color.lambda.ts

```diff
@@ -25,20 +35,37 @@ export const getAppThemeColorLambda = (theme?: AppTheme) => {
     }
 
   // by default halloween as only two options for now:
+  // return {
+  //   light: [
+  //     Halloween.one,
+  //     Halloween.two,
+  //     Halloween.three,
+  //     Halloween.four,
+  //     Halloween.four,
+  //   ],
+  //   dark: [
+  //     Halloween.one,
+  //     Halloween.two,
+  //     Halloween.three,
+  //     Halloween.four,
+  //     Halloween.four,
+  //   ],
+  // }
+
```

Comment:
Use something like
`if (theme === AppTheme.Christmas) { ... }`

Reply:
Handled in [8ef685e](https://github.com/ajktown/ConsistencyGPT/pull/74/commits/8ef685e54a10fbb3a6fff4f9964b248e91c083eb)